content/en/agent/guide/network.md

```diff
@@ -145,6 +145,18 @@ Open the following ports in order to benefit from all the Agent functionalities:
 
 For a detailed configuration guide on proxy setup, see [Agent Proxy Configuration][8].
 
+## Agent-side data buffering on network unavailability
```

Comment:
can we please elaborate what type of traffic is stored by the Agent, specifically -- does it include:
1. Logs -- if yes, let's also add a link from this [guide](https://docs.datadoghq.com/logs/faq/log-collection-is-the-datadog-agent-losing-logs/#pagetitle) to this page
2. Metrics
3. Traces?

Reply:
The traffic stored on disk includes only the metrics. Doc updated.